src/com/javafee/task/bankaccount/BankAccountPremium.java

```diff
@@ -0,0 +1,10 @@
+package src.com.javafee.task.bankaccount;
+
+public class BankAccountPremium extends BankAccount{
+
+    void withdraw(double withdraw_anmount){
```

Comment:
As per the business requirements, the client should get the return once withdrawing the % of the balance.